server/spawn-vehicle.lua

```diff
@@ -22,6 +22,13 @@ lib.callback.register('qbx_garages:server:spawnVehicle', function (source, vehic
     end
     local garageType = GetGarageType(garageName)
 
+    local spawnCoords = accessPoint.spawn or accessPoint.coords
+    local nearbyVehicle = lib.getClosestVehicle(vec3(spawnCoords.x, spawnCoords.y, spawnCoords.z), 5.0, false)
```

Comment:
Can you pull the distance check variable out to the config. Depending on where folks place these spots 5.0 might not work for them.